### PR TITLE
Default properties

### PR DESCRIPTION
**Pull request by ale-kacperczyk**
## 🚀 Links

- [Feature branch](https://developers.labs.livechat.com/docs/feature/default-properties)
- [Jira](https://livechatinc.atlassian.net/browse/DPS-2048)

## 📓 Description
New document, Default properties in Messaging > References > Default properties.

### Content

- New content

### CODE REVIEW

File: content/messaging/references/default-properties/index.mdx [hunk 1]
```diff
@@ -0,0 +1,495 @@
...
+Specifies if the continuous messaging mode is enabled.
+
+|               |                                                   |
+| ------------- | ------------------------------------------------- |
+| **Access**    | `r` for Agents and Customers in chats and threads |
+| **Data type** | `bool`                                            |
+
+### email_follow_up
+
+Specifies if there should be an email follow-up of the message left by the Customer. For example, when a Customer left a message because they were no online Agents.
```
Comment (litwinow) [inline content/messaging/references/default-properties/index.mdx:196]: I think it'd be nice to add that the email will be sent if messaging mode is enabled on the license

File: content/messaging/references/default-properties/index.mdx [hunk 2]
```diff
@@ -0,0 +1,495 @@
...
+| **Data type** | `string`                                                                           |
+| **Values**    | `prechat` `postchat`                                                               |
+
+### greeting_id
+
+Specifies the ID of the greeting configured within a license. [More on greetings...](/messaging/customer-chat-api/#accept-greeting)
+
+|               |                                                                                                                 |
+| ------------- | --------------------------------------------------------------------------------------------------------------- |
+| **Access**    | <ul><li>for Agents, `r` in threads and events</li><li>for Customers, `w` in threads and `r` in events</li></ul> |
```
Comment (litwinow) [inline content/messaging/references/default-properties/index.mdx:60]:
```suggestion
| **Access**    | <ul><li>for Agents, `r` in threads and events</li><li>for Customers, `rw` in threads and `r` in events</li></ul> |
```

File: content/messaging/references/default-properties/index.mdx [hunk 3]
```diff
@@ -0,0 +1,495 @@
...
+| **Data type** | `int`                                                                                                           |
+
+### greeting_unique_id
+
+Specifies the ID of a particular greeting instance sent to the Customer. [More on greetings...](/messaging/customer-chat-api/#accept-greeting)
+
+|               |                                                                                                                 |
+| ------------- | --------------------------------------------------------------------------------------------------------------- |
+| **Access**    | <ul><li>for Agents, `r` in threads and events</li><li>for Customers, `w` in threads and `r` in events</li></ul> |
```
Comment (litwinow) [inline content/messaging/references/default-properties/index.mdx:69]:
```suggestion
| **Access**    | <ul><li>for Agents, `r` in threads and events</li><li>for Customers, `rw` in threads and `r` in events</li></ul> |
```

File: content/messaging/references/default-properties/index.mdx [hunk 4]
```diff
@@ -0,0 +1,495 @@
...
+Specifies the ID of the welcome message author.
+
+|               |                                         |
+| ------------- | --------------------------------------- |
+| **Access**    | `rw` for Customers in chats and threads |
+| **Data type** | `string`                                |
+
+### welcome_message
+
+Specifies if a welcome message was sent to the Customer.
```
Comment (litwinow) [inline content/messaging/references/default-properties/index.mdx:92]: it specifies that given event is a welcome message

File: content/messaging/references/default-properties/index.mdx [hunk 5]
```diff
@@ -0,0 +1,495 @@
...
+
+The properties describing chat rating.
+
+### comment
+
+Contains the text message rating left by the Customer.
+
+|               |                                                                     |
+| ------------- | ------------------------------------------------------------------- |
+| **Access**    | `r` for Agents and `rw` for Customers in chats, threads, and events |
```
Comment (litwinow) [inline content/messaging/references/default-properties/index.mdx:147]: it's inconsistent with other props, they use separate bullet points for agents and customers

File: content/messaging/references/default-properties/index.mdx [hunk 6]
```diff
@@ -0,0 +1,495 @@
...
+</Section>
+
+<Section>
+<Text>
+
+## routing
+
+### continuous
+
+Specifies if the continuous messaging mode is enabled.
```
Comment (litwinow) [inline content/messaging/references/default-properties/index.mdx:187]: Hmm I think it'd be nice to underline that this property can be different across different chats.
Messaging Mode is a product feature but any integration can create both continuous and non-continuous chats despite of Messaging Mode (eg. facebook chats are always continuous even if the license doesn't use Messaging Mode feature).

File: content/messaging/references/default-properties/index.mdx [hunk 7]
```diff
@@ -0,0 +1,495 @@
...
+| **Access**    | <ul><li>for Agents, `r` in chats</li> <li>for Customers, `rw` in chats </li></ul> |
+| **Data type** | `bool`                                                                            |
+
+### idle
+
+Specifies if a chat is idle or active.
+
+|               |                |
+| ------------- | -------------- |
+| **Access**    | `r` for Agents |
```
Comment (litwinow) [inline content/messaging/references/default-properties/index.mdx:209]:
```suggestion
| **Access**    | `r` for Agents in threads |
```

sorry, it wasn't specified on the branch I sent you :(

File: content/messaging/references/default-properties/index.mdx [hunk 8]
```diff
@@ -0,0 +1,495 @@
...
+  "properties": {
+      "routing": {
+          "continuous": true,
+          "email_follow_up": false,
+          "idle": false,
+          "last_transfer_timestamp": 83765782567,
+          "pinned": false,
+          "referrer": "https://livechatinc.com/features",
+          "start_url": "https://livechatinc.com",
+          "transcript_email": "t.anderson@gmail.com"
```
Comment (litwinow) [inline content/messaging/references/default-properties/index.mdx:281]:
```suggestion
          "transcript_email": "t.anderson@example.com"
```

Comment (ale-kacperczyk) [reply content/messaging/references/default-properties/index.mdx:281]: nice catch

**Pull request merged**